wxMaxima.yaml

```diff
@@ -0,0 +1,31 @@
+name: wxMaxima
+buildsystem: cmake-ninja
+builddir: true
+sources:
+  - type: archive
+    url: https://github.com/wxMaxima-developers/wxmaxima/archive/Version-18.12.0.tar.gz
+    sha256: 7098bacce2023bf0c14f17f6bc5368d0732770437e5cff3ef4d65b026f432e5c
+  - type: patch
+    path: wxMaxima-18.12.0-version-fix.patch
+  - type: file
+    path: wxMaxima.appdata.xml
+    dest: data
+  - type: shell
+    commands:
+      - desktop-file-edit --set-key="Terminal" --set-value="false" "data/wxMaxima.desktop";
+      - desktop-file-edit --set-key="StartupNotify" --set-value="true" "data/wxMaxima.desktop";
+      - desktop-file-edit --set-key="StartupWMClass" --set-value="wxmaxima" "data/wxMaxima.desktop";
+post-install:
+  - icon_in="../data/wxmaxima.svg";
+    icon_out="wxmaxima.png";
+    for s in {16,22,24,32,48,64,72,96,128,192,256,512}; do
+    [[ ! -f "${FLATPAK_DEST}/share/icons/hicolor/${s}x${s}/apps/${icon_out}" ]] || continue;
+    rsvg-convert "${icon_in}" -w "${s}" -h "${s}" -a -f png -o "${icon_out}";
+    install -p -D -m 0644 "${icon_out}" -t "${FLATPAK_DEST}/share/icons/hicolor/${s}x${s}/apps/";
+    done;
+  - install -p -D -m 0644 "../COPYING" -t "${FLATPAK_DEST}/share/licenses/wxmaxima/";
```

Comment:
The files isn't exported, it's part of Sources extension.

Reply:
Done.

https://github.com/flathub/flathub/pull/801/commits/de93f9e911a070beb1913aade48a4b7f544a2eb9#diff-a5c95a1d14fc9cd7d409e1ea4fbfa63cL26